extensions/wrapper/wrapper/src/test/java/de/sovity/edc/ext/wrapper/api/ui/pages/catalog/CatalogApiTest.java

```diff
@@ -2,12 +2,15 @@
 
```

Comment:
Missing copyright header. This time no suggestion because GH doesn't let me select the first line...